tests/bson/bson-javascript-set_state-001.phpt

```diff
@@ -31,39 +31,39 @@ echo "\n\n";
 <?php exit(0); ?>
 --EXPECTF--
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo(bar) { return bar; }',
-%w'scope' => NULL,
+   'code' => 'function foo(bar) { return bar; }',
+   'scope' => NULL,
 ))
 
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo(bar) { return bar; }',
-%w'scope' => 
-  %Sarray(
-  %S),
+   'code' => 'function foo(bar) { return bar; }',
+   'scope' =>%w
+  %r\(object\)? %rarray(
```

Comment:
Why did you add `%r\(object\)? %r` here but not below, where `%Sarray` remains?

Reply:
I missed that one. I replaced other occurrences of `%Sarray` with this as it's a more precise match.